Gruntfile.js

```diff
@@ -170,14 +170,18 @@ module.exports = function(grunt) {
   // Former grunt-pot
   grunt.registerTask('pot', ['shell:pot']);
 
-  // Build assets, docs and language files
-  grunt.registerTask('build', 'Build assets, docs and language files', [
-    'less',
-    'cssmin',
-    'uglify',
```

Comment:
I don't disagree with making a separate `UI` task to separately run the less/cssmin/uglify items. It's nice to be able to run those separately. Whether or not that's done, these items should remain within the `build` task so that the release process remains as simple as possible: https://github.com/WPBuddy/largo/issues/1798